### ISSUE TITLE
Create Error pages for EBI site

### ISSUE THREAD

**Issue by sandykadam** [ebiwd/EBI-Corporatesite#26]
Hi @khawkins98,

Can we please have error pages similar like we have done for embl.org 500, 403 etc and also have under-maintenance page. We could use these in different scenarios.

Thanks

**Comment by khawkins98** [comment]
This should be pretty easy to do.

As for reusing the embl.org ones, obviously not hard to copy the HTML around. In #24 I've also thought about moving this project to 11ty for:

- consistency with other projects
- would be good to have a template engine for these pages
- would make it easier to reuse code from embl.org static-html-pages

**Comment by sandykadam** [comment]
I just noticed there is another repo for EBI error pages https://gitlab.ebi.ac.uk/ebiwd/errorpages but we can get rid of this once we have errorpages in this Github main repo.

**Comment by khawkins98** [comment]
Yeah, would be good to get the error pages off of the old compliance look:

https://www.ebi.ac.uk/adfads

![image](https://user-images.githubusercontent.com/928100/110144358-8a33f180-7dd8-11eb-8d73-3680d1048e63.png)
